tensortrade/exchanges/exchange.py

```diff
@@ -185,6 +185,10 @@ def has_next_observation(self) -> bool:
         """
         raise NotImplementedError
 
+    @property
+    def commission_percent(self):
+        return self._commission_percent / 100
```

Comment:
This is unintuitive. Why would `commission_percent` should be equal to `_commission_percent`.

The property should be renamed to `commission_amount`, since it is no longer denoted in %.

Reply:
Agreed. Will change.